### ISSUE TITLE
How to Access your Google Drive with Globus - suggested edits

### ISSUE THREAD

**Issue by ruthaydt** [globusonline/docs.globus.org#176]
This feedback is based on document currently on staging.   Will put new comment per recommended change.

**Comment by teresas** [comment]
I'm not sure why this issue exists. Can you clarify if there are any changes to the How To?

**Comment by ruthaydt** [comment]
@teresas -- Sorry.   I thought there would be a few updates that I would put here, but as I got into the doc I had more extensive suggestions to change it from embedded section to "How To" format, and also to clarify things I don't understand.   

 Ended up creating a working Google Doc to hash through & currently waiting for some help from experts to fill in some of my knowledge gaps.  The google doc is here:https://docs.google.com/a/globus.org/document/d/1UJyULbg2fe2RluyuUTl7oma1kTpTe59Dn_8k_yEgSLE/edit?usp=sharing

If you want to close this and I'll open another when things settle down, that's fine with me.
